resources/js/components/slideovers/common/FreezeSlideover.vue

```diff
@@ -121,11 +130,14 @@ const tokenId = ref({
     tokenId: props.item?.tokenId ?? '',
 });
 const collectionAccount = ref('');
+const freezeState = ref(FreezeStateType.PERMANENT);
```

Comment:
Idk if that's the default value. But the default should be temporary not permanent.

Reply:
Thank you